Include/object.h

```diff
@@ -643,6 +764,19 @@ static inline Py_ALWAYS_INLINE void Py_INCREF(PyObject *op)
 #  define Py_INCREF(op) Py_INCREF(_PyObject_CAST(op))
 #endif
 
+
+#if !defined(Py_LIMITED_API) && defined(Py_NOGIL)
+// Implements Py_DECREF on objects not owned by the current thread.
+PyAPI_FUNC(void) _Py_DecRefShared(PyObject *);
+PyAPI_FUNC(void) _Py_DecRefSharedDebug(PyObject *, const char *, int);
+
+// Called from Py_DECREF by the owning thread when the local refcount reaches
+// zero. The call will deallocate the object if the shared refcount is also
+// zero. Otherwise, the thread gives up ownership and merges the reference
+// count fields.
+PyAPI_FUNC(void) _Py_MergeZeroRefcount(PyObject *);
```

Comment:
When I read the code this function is handling the object when the local ref count is zero to decide dealloc or not.
So might be, we can improve function name to understand what's happening
```suggestion
PyAPI_FUNC(void) _Py_MergeZeroRefLocalcount(PyObject *);
```

or 

```suggestion
PyAPI_FUNC(void) _Py_MergeZeroLocalRefcount(PyObject *);
```

Reply:
I've named it `_Py_MergeZeroLocalRefcount`